src/Spryker/Client/RabbitMq/RabbitMqConfig.php

```diff
@@ -103,7 +103,7 @@ public function getDefaultQueueConnectionConfig(): QueueConnectionTransfer
             ->setConnectionTimeout(static::AMQP_STREAM_CONNECTION_CONNECTION_TIMEOUT)
             ->setReadWriteTimeout(static::AMQP_STREAM_CONNECTION_READ_WRITE_TIMEOUT)
             ->setKeepAlive(static::AMQP_STREAM_CONNECTION_KEEP_ALIVE)
-            ->setHeartBeat(static::AMQP_STREAM_CONNECTION_HEART_BEAT)
```

Comment:
I think we have to deprecate + comment on this variable, since it's no longer used.

Actually, it might be also a potential BC-break.

Reply:
fixed as you suggested